gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/tools/internal/ToolsBase.java

```diff
@@ -269,12 +270,6 @@ protected void addConnectionOptions(final Options opts) {
             "TOOLS_MCAST_ADDRESS_MESSAGE")).create(MCAST_ADDRESS);
     opts.addOption(opt);
 
-    opt = new GfxdOptionBuilder().withArgName(LocalizedResource.getMessage(
-        "TOOLS_LOCATORS_ARG")).hasArg().withValueSeparator('=')
-        .withDescription(LocalizedResource.getMessage(
-            "TOOLS_LOCATORS_MESSAGE")).create(LOCATORS);
-    opts.addOption(opt);
-
```

Comment:
also move TOOLS_MCAST_PORT_MESSAGE, TOOLS_MCAST_ADDRESS_MESSAGE, TOOLS_BIND_ADDRESS_MESSAGE and TOOLS_AUTH_PROVIDER_MESSAGE into additional options since these are only for embedded mode connections which are not supported anymore

Reply:
Moved